packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.4+2
+
+* Fixes to enable compilation to WebAssembly. 
```

Comment:
It seems like this should say "Fixes pub.dev detection of WebAssembly support." (or something like that) instead, since per discussion in Discord this package compiles fine for wasm without this PR.